### ISSUE TITLE
Multiple locations per document

### ISSUE THREAD

**Issue by ayastreb** [ayastreb/jekyll-maps#6]
As a user I should be able to provide multiple locations per single document.
For example:

``` yaml
location:
  - title: point A
    latitude:  43.988223
    longitude: 13.223221
  - title: point B
    latitude:  45.987348
    longitude: 5.4424479
    url: /somewhere-else
```

should produce two points on the map with this document.
If no URL provided for specifi point, document's main URL is used.

**Comment by jolesbi** [comment]
+1

**Comment by ayastreb** [comment]
Implemented in 2.0.4
